test/TestLogger.cs

```diff
@@ -1,8 +1,12 @@
+ 
+
 using Microsoft.Extensions.Logging;
 
+namespace HealthChecks.ApplicationStatus.Tests;
```

Comment:
No. Please revert. This file is by design has no namespace and is used by **each** test project.

Reply:
The same for `TestLoggerProvider`.